test_runner/batch_others/test_tenant_conf.py

```diff
@@ -34,15 +34,15 @@ def test_tenant_config(zenith_env_builder: ZenithEnvBuilder):
             pscur.execute(f"show {env.initial_tenant.hex}")
             res = pscur.fetchone()
             log.info(f"initial_tenant res: {res}")
-            assert res == (10000, 1048576, 1, 10, 67108864, 100, 2592000)
+            assert res == (10000, 1048576, 1, 10, 67108864, 100, 3, 2592000)
 
     # check the configuration of the new tenant
     with closing(env.pageserver.connect()) as psconn:
         with psconn.cursor() as pscur:
             pscur.execute(f"show {tenant.hex}")
             res = pscur.fetchone()
             log.info(f"res: {res}")
-            assert res == (20000, 1048576, 1, 10, 67108864, 30, 2592000)
```

Comment:
Can we refactor these tests, so that they don't need to be updated every time we add a new pageserver config option?

Reply:
Ya `gc_threshold` is a bit misleading. Fixed below, I'll squash these before merging.